tls-2pc-core/src/prf/mod.rs

```diff
@@ -1,11 +1,82 @@
+//! This module contains the protocol for computing TLS SHA-256 HMAC PRF using 2PC in such a way
```

Comment:
Can we add a step-by-step description or a link thereto similar to https://github.com/tlsnotary/circuits/blob/9f6864934b2be9a5745c96fb40f40dfe3b6f690f/README#L35
?